aviary/examples/run_new_detailed_takeoff.py

```diff
@@ -0,0 +1,320 @@
+                     {'method': 'low_speed',
+                      'ground_altitude': 0.,  # units='ft'
+                      'angles_of_attack': [
+                          0.0, 1.0, 2.0, 3.0, 4.0, 5.0,
+                          6.0, 7.0, 8.0, 9.0, 10.0, 11.0,
+                          12.0, 13.0, 14.0, 15.0],  # units='deg'
+                      'lift_coefficients': [
+                          0.5178, 0.6, 0.75, 0.85, 0.95, 1.05,
+                          1.15, 1.25, 1.35, 1.5, 1.6, 1.7,
+                          1.8, 1.85, 1.9, 1.95],
+                      'drag_coefficients': [
+                          0.0674, 0.065, 0.065, 0.07, 0.072, 0.076,
+                          0.084, 0.09, 0.10, 0.11, 0.12, 0.13,
+                          0.15, 0.16, 0.18, 0.20],
+                      'lift_coefficient_factor': 1.,
+                      'drag_coefficient_factor': 1.}}
+
+optimize_mach = False
+optimize_altitude = False
+
+# Currently the more complete takeoff trajectory requires SNOPT to converge.
+# If False, the takeoff trajectory will be simplified to a ground roll only
+# to ensure the methods_for_level2 API works.
+use_full_takeoff = False
+
+phase_info = {
+    "pre_mission": {"include_takeoff": False, "optimize_mass": False},
+    'AB': {
+        'user_options': {
+            'num_segments': 5,
+            'order': 3,
+            'fix_initial': True,
+            'ground_roll': True,
+            'duration_ref': (100., 'kn'),
+            'duration_bounds': ((100., 500.), 'kn'),
+        },
+        'subsystem_options': subsystem_options,
+        'initial_guesses': {
+            'distance': [(0., 2.e3), 'ft'],
+            'time': [(0., 20.), 's'],
+            'velocity': [(1., 120.), 'kn'],
+            'mass': [(175.e3, 174.85e3), 'lbm'],
+        },
+    },
+    'rotate': {
+        'user_options': {
+            'num_segments': 5,
+            'order': 3,
+            'fix_initial': False,
+            'throttle_setting': throttle_max,
+            'ground_roll': True,
+            'clean': False,
+            'initial_ref': (1.e3, 'ft'),
+            'initial_bounds': ((1.e3, 3.e3), 'ft'),
+            'duration_ref': (1.e3, 'ft'),
+            'duration_bounds': ((200., 2.e3), 'ft'),
+            'mach_bounds': ((0.18, 0.2), 'unitless'),
+            'control_order': 1,
+            'opt': True,
+            'balance_throttle': False,
+            'optimize_mach': optimize_mach,
+            'optimize_altitude': optimize_altitude,
+            'rotation': True,
+            'constraints': {
+                'normal_force': {
+                    'equals': 0.,
+                    'loc': 'final',
+                    'units': 'lbf',
+                    'type': 'boundary',
+                    'ref': 10.e5,
+                },
+            },
+        },
+        'subsystem_options': subsystem_options,
+        'initial_guesses': {
+            'distance': [(2.e3, 1.e3), 'ft'],
```

Comment:
Does distance really go down in this phase?

Reply:
I think this is one of the instances where the second value is the delta instead of the final

Reply:
oh yeah, i have some vague memory of encountering this before when distance was the integration variable.

Reply:
Carl's right; Ken we can and should clean this up in general, here's the previous issue: https://github.com/OpenMDAO/Aviary_deprecated/issues/388

I won't want to modify this in this PR though.